### ISSUE TITLE
King Solomons Bracelets

### ISSUE THREAD

**Issue by LordChanning** [aikaterna/gobcog#245]
![Screenshot (4079)](https://user-images.githubusercontent.com/39933493/87116912-5b0eb580-c234-11ea-92d3-cdb4f6ab82c5.png)
![Screenshot (4080)](https://user-images.githubusercontent.com/39933493/87116914-5ba74c00-c234-11ea-9ae3-74abf881f18d.png)
![Screenshot (4081)](https://user-images.githubusercontent.com/39933493/87116915-5ba74c00-c234-11ea-92da-5e814606f6ec.png)
![Screenshot (4082)](https://user-images.githubusercontent.com/39933493/87116916-5ba74c00-c234-11ea-93df-ec994a6bb487.png)

The item King Solomons Bracelets show correctly in the setinfo and loadout show screens but is missing an n and has 0 stats in the stats and backpack. I'm using a custom theme but the tr_set file matches exactly

**Comment by LordChanning** [comment]
![Screenshot (4084)](https://user-images.githubusercontent.com/39933493/87117678-64991d00-c236-11ea-8123-a8617cf7a89e.png)

Same thing on default theme too

**Comment by Drapersniper** [comment]
@LordChanning  what is your adventure version. What is the output of `[p]adventureset version`  if it is not 3.2.30 you are outdated.

**Comment by LordChanning** [comment]
I am on version 3.2.31

**Comment by Drapersniper** [comment]
Unfortunately the fix for this is not feasible.

This originated due to you having a custom theme that mirrors the default theme, which is not safe to do as set items are not supposed to work cross theme. 

Recently a typo in that item name was fixed, and 3.2.30 had a migration fix to mitigate that typo. The migration was completed, however due to your custom theme not having the typo fix players were still distributed the item with the typo, which will always have 0 stats and will not contribute to any set bonus.

In other words the item with the typo from your custom theme is paperweight
